src/Shared/SFA.DAS.SharedOuterApi/InnerApi/Responses/RequestApprenticeTraining/GetSettingsResponse.cs

```diff
@@ -0,0 +1,11 @@
+﻿using System.Diagnostics.CodeAnalysis;
+
+namespace SFA.DAS.SharedOuterApi.InnerApi.Responses.RequestApprenticeTraining
+{
+    [ExcludeFromCodeCoverage]
+    public class GetSettingsResponse 
+    {
+        public int ExpiryAfterMonths { get; set; }
+        public int RemovedAfterExpiryMonths { get; set; }
+    }
```

Comment:
Should this also include ProviderRemovedAfterRequestedMonths ?

Reply:
Yes it should as this is the shared response

Reply:
Done